### DISCUSSION TITLE
如何幫助蕭煌奇開車或感知這個世界?

### DISCUSSION THREAD

**Discussion by KozakHou** [1101-GS4719-C/Python-Community-GS4719-C#1282] [General]
[大家是否最近有看到蕭煌奇FB的貼文問說要不要搭上他的車?](https://www.facebook.com/RickyHsiao)

大眾普遍認為是臺末班車。

### 那麼有沒有方法幫助他呢?

以下提供一個解法
使用智慧眼鏡搭配OpenCV套件，每5秒截圖，並使用Image Caption的技術(Image --> Text)，再使用TTS(Text to Speech)的模型，讓人能夠知道前方的情境是如何。

情景: 蕭先生在7-11買東西，但是不知道現在是在哪個商品區前，所以微處理器收到截圖後，轉成文字再轉成語音，讓蕭先生知道他再準備買思樂冰。

目前有Gupta, S. (2021), Shubham et al., n.d. (2021)等提出CNN-LSTM-TTS的架構去實現，建立一個視覺與聽覺溝通的橋樑。
我最近想嘗試用CNN-Transformer-TTS來實作，2個月內應該可以有成果。

沒有的話，就來當下學期的AI專題

References: 

Gupta, S. (2021) ‘Step by Step Guide to Build Image Caption Generator using Deep Learning’, Analytics Vidhya, 18 December. Available at: https://www.analyticsvidhya.com/blog/2021/12/step-by-step-guide-to-build-image-caption-generator-using-deep-learning/ (Accessed: 24 April 2023)

Shubham, R., Megha, G., Krishna, S., Paras, S., Shraddha, K., Deepali, P., n.d. IMAGE CAPTIONING GENERATOR SYSTEM WITH CAPTION TO SPEECH CONVERSION MECHANISM. Int. Res. J. Mod. Eng. Technol. Sci.

**Comment by Jack-1D** [comment]
這篇文章相當有趣。可以增加視障者的自主性，讓他們不需要因為每件事都需要他人協助而感到自卑。但我認為當初蕭煌奇問大家要不要搭的應該是他弟蕭煌車:sweat_smile:
